app/views/suppliers/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<p id="notice"><%= notice %></p>
+
+<h1 align="center">Suppiler</h1>
+
+<table border="1" align="center" cellspasing="5" cellpadding="5">
+  <thead>
+    <tr>
+      <th>supplier_name</th>
```

Comment:
why are you using table heading name with underscore? are you copying table field name directly here?